metal/mmtl/task.py

```diff
@@ -79,7 +79,7 @@ def __init__(
         head_module,
         scorer=Scorer(standard_metrics=[]),
         loss_hat_func=(
-            lambda Y_prob, Y_gold: F.mse_loss(torch.sigmoid(Y_prob), Y_gold)
+            lambda Y_prob, Y_gold: F.mse_loss(torch.sigmoid(Y_prob), Y_gold.float())
```

Comment:
@bhancock8 should this be fixed somewhere else?  I was getting a weird long-float error and just casted it away...

Reply:
Yeah...suspicious this hasn't come up elsewhere before...can you mark this with a TODO or HACK note so we know to pay attention to it?